recipes/barrnap/0.8/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "barrnap" %}
+{% set version = "0.8" %}
+{% set sha256 = "82004930767e92b61539c0de27ff837b8b7af01236e565f1473c63668cf0370f"%}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}_v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  url: https://github.com/tseemann/{{ name }}/archive/{{ version }}.tar.gz
+
+build:
+  number: 0
+
+requirements:
+  build: []
+  run:
+    - perl >=5.22.0
+    - hmmer >=3.1b
+    - libgcc
```

Comment:
I assume it was originally in there because Barrnap upstream includes the `nhmmer` binaries for osx and linux and they were just copied into the conda package.